src/main/java/com/conveyal/r5/profile/PerTargetPropagater.java

```diff
@@ -35,15 +35,16 @@
     /** the profilerequest (used for walk speed etc.) */
     public final ProfileRequest request;
 
-    public PerTargetPropagater (int[][] travelTimesToStopsEachIteration, int[] nonTransferTravelTimesToTargets, LinkedPointSet targets, ProfileRequest request, int cutoffSeconds) {
+    public PerTargetPropagater (int[][] travelTimesToStopsEachIteration, int[] nonTransitTravelTimesToTargets, LinkedPointSet targets, ProfileRequest request, int cutoffSeconds) {
         this.travelTimesToStopsEachIteration = travelTimesToStopsEachIteration;
-        this.nonTransferTravelTimesToTargets = nonTransferTravelTimesToTargets;
+        this.nonTransitTravelTimesToTargets = nonTransitTravelTimesToTargets;
         this.targets = targets;
         this.request = request;
         this.cutoffSeconds = cutoffSeconds;
     }
 
-    public void propagate (Reducer reducer) {
+    private void propagate (Reducer reducer, TravelTimeReducer travelTimeReducer) {
```

Comment:
Oh hey, you already did that below. Nice. But a line of Javadoc explaining this is the internal interface for those two external interfaces and which parameter takes priority would be good.